### PR TITLE
Navigation to Homepage by clicking on logo 

### PR DESCRIPTION
**Pull request by itsPriyanshuhere**
## Description 📝

fix issue number #247
Any click on the logo would navigate the user to the home page. Hover effect was added which changes the cursor to points as the user hovers over the logo.

## Changes 💬

Used react router to navigate by pushing "/"  using router.push("/").
Used tailwind css to convert the cursor to pointer as user hovers over it.

## Checklist ✅

- [x] Reviewed the code changes thoroughly.
- [x] Tested the changes locally and ensured they work as expected.
- [ ] Updated relevant documentation.
- [ ] Added appropriate test cases, if applicable.
- [ ] Considered potential security or performance implications.
- [x] Code adheres to the project's style guidelines.
- [x] Addressed all reviewer comments and concerns, if any.

## Additional Notes 📒

This would make the navigation easy for user if they want to redirect to home page.

### CODE REVIEW

File: Components/LogoWithName.tsx [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+
+  const router = useRouter();
+
+  const handleClickonLogo = () => {
+    router.push("/");
+  };
+
   return (
-    <div className="flex  items-center   w-full h-fit  justify-center ">
+    <div className="flex  items-center   w-full h-fit  justify-center cursor-pointer " onClick={handleClickonLogo}>
```
Comment (lalitmee) [inline Components/LogoWithName.tsx:13]: Please put proper html attributes as we are putting `onClick` on a `div`.

**Review by lalitmee — CHANGES_REQUESTED** [review]
Revert `package-lock.json` changes. Those are not required.

**Comment by PiyushKalyanpy** [comment]
Hey there, 
We are closing this pull request as the changes we requested were not implemented. Please create a new PR once you implement the changes.

**Comment by PiyushKalyanpy** [comment]
Hey there, 
We are closing this pull request as the changes we requested were not implemented. Please create a new PR once you implement the changes.

**Pull request closed without merging**